semgrep/semgrep/target_manager.py

```diff
@@ -171,7 +175,11 @@ def _find_files_with_extention(
             """
             Return set of all files in curr_dir with given extension
             """
-            return set(p for p in curr_dir.rglob(f"*{extension}") if p.is_file())
+            return {
```

Comment:
Should we apply the same logic to the `_parse_output` loop above? That way we have the same behavior between git tracked files and not.

Also, should we lift `is_file` into `_is_valid`?

Reply:
There's another branch that follows directories, so we don't want to include `is_file`